source/compiler/qsc_frontend/src/resolve/tests.rs

```diff
@@ -3029,16 +3029,17 @@ fn check_locals(input: &str, expect: &Expect) {
 
     let locals = locals.get_all_at_offset(cursor_offset);
     let actual = locals.iter().fold(String::new(), |mut output, l| {
-        let _ = writeln!(
-            output,
-            "{} ({})",
-            l.name,
-            match l.kind {
-                LocalKind::Item(item_id) => item_id.to_string(),
-                LocalKind::TyParam(param_id) => format!("ty_param {param_id}"),
-                LocalKind::Var(node_id) => format!("var {node_id}"),
-            }
-        );
+        let _ = match l {
```

Comment:
I'm not seeing any tests in this file that make use of this new printing behavior. Could you add a resolver test that resolves an alias, which should creates a different in printed output other than "namespace 3"?